File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/2949.cs
using System;

class URI
{
    static void Main() {
        int n, i, a=0, elf=0, h=0, m=0, x=0;
        string nome, e;
        n = int.Parse(System.Console.ReadLine().Trim());
        for(i=0;i<n;i++){
            nome = System.Console.ReadLine().Trim();
            string[] linha = nome.Split(' ');
            e = linha[1];
            if(e=="A") a++;
            else if(e=="E") elf++;
            else if(e=="H") h++;
            else if(e=="M") m++;
            else x++;
        }
        Console.Write("{0} Hobbit(s)\n", x);
        Console.Write("{0} Humano(s)\n", h);
        Console.Write("{0} Elfo(s)\n", elf);
        Console.Write("{0} Anao(s)\n", a);
        Console.Write("{0} Mago(s)\n", m);
    }
}

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/2949.lua
a=0
elf=0
h=0
m=0
x=0

n = io.read("*number")
for i=1,n do
    nome = io.read("*line")
    e = string.sub(nome, string.len(nome), string.len(nome))
    if(e=='A') then 
        a=a+1
    elseif(e=='E') then 
        elf=elf+1
    elseif(e=='H') then 
        h=h+1
    elseif(e=='M') then 
        m=m+1
    else 
        x=x+1
    end
end
print(x.." Hobbit(s)")
print(h.." Humano(s)")
print(elf.." Elfo(s)")
print(a.." Anao(s)")
print(m.." Mago(s)")
